git_theta/checkpoints.py

```diff
@@ -137,7 +137,7 @@ def _iterate_dir_leaves(root, prefix):
     return _iterate_dir_leaves(root, [])
 
 
-def get_checkpoint(checkpoint_type: str) -> Checkpoint:
+def get_checkpoint_class_by_name(checkpoint_type: str) -> Checkpoint:
```

Comment:
So technically the checkpoint handler doesn't have to be a class, the entry point syntax (`name = package.module:thing`) can refer to functions, vars, classes, etc. as `thing`. We assume it is a class def but it could be anything.

I think if we want a better name something like `get_checkpoint_handler` might make more sense?

Reply:
Agreed